plugins/item_tasks/web_client/models/WidgetModel.js

```diff
@@ -302,6 +305,8 @@ var WidgetModel = Backbone.Model.extend({
             return 'Value must be a folder';
         } else if (this.get('type') === 'new-file' && type !== 'folder') {
             return 'Value must be a folder';
+        } else if (this.get('type') === 'new-directory' && type !== 'folder') {
```

Comment:
What if we want to create the new folder at the top level, directly inside a collection (or a user's home directory).